junifer/storage/base.py

```diff
@@ -51,9 +51,8 @@ def get_meta(self) -> Dict:
         }
         return meta
 
-    # TODO: is raising ValueError required?
     @abstractmethod
-    def validate(self, input_: List[str]) -> bool:
```

Comment:
It still should be the same: `validate`. However, the storage has no output.

Reply:
So it raises an error if `validate` fails?

Reply:
From what I see, `validate` in `SQLiteFeatureStorage` returns a bool.

Reply:
It should not. It should raise an error if it fails to validate the action.

Reply:
Okay then I'll make the changes.